Paybox/System/Base/Request.php

```diff
@@ -147,10 +148,23 @@ public function getForm($options = array())
         $options['csrf_protection'] = false;
 
         $parameters = $this->getParameters();
-        $builder = $this->factory->createNamedBuilder('', 'form', $parameters, $options);
-
-        foreach ($parameters as $key => $value) {
-            $builder->add($key, 'hidden');
+        // If symfony version is 3.* then we use the FQCN for form types
+        // Else we use the IDs.
+        if (substr(Kernel::VERSION, 0, 1) === "3") {
```

Comment:
Sorry but this is wrong. For the proof, see my composer.json:

```json
{
    "name": "root/compo",
    "require": {
        "symfony/http-kernel": "^3.0",
        "symfony/form": "2.3.*"
    },
    "authors": [
        {
            "name": "Sullivan SENECHAL",
            "email": "soullivaneuh@gmail.com"
        }
    ]
}
```

Try a `composer update` on this one, and you will see the installed packages: 

```
symfony/form              v2.3.39 Symfony Form Component
symfony/http-kernel       v3.0.4  Symfony HttpKernel Component
```

So you can install kernel 3.0 and form 2.3 without any problem.

And BTW, class name usage start on 2.8, not 3.0. The deprecated error is use to prepare developer to use the next major. So it should be fix for 2.8 also and remove deprecation notices.

See #72 for the fix.